### PR TITLE
Add img alt for static pages

### PR DESCRIPTION
**Pull request by batpigandme**
* Add alt text for images on: packages, learn, and contribute pages

### CODE REVIEW

File: content/help.md [hunk 1]
```diff
@@ -57,16 +57,16 @@ mean(y)
...
+<img src="../reprex-addin.png" alt="reprex addin interface"></img>
 
 In either case, you can eventually [explore other features](http://reprex.tidyverse.org), such as formatting output for Stack Overflow or as a commented R script. reprex even uploads figures so you can easily ask questions about ggplot2.
 
 If your code is not self-contained, running `reprex()` results in an error. It may feel like tough love, but this way you can get your story straight in private. The reprex format also strongly encourages you to find the minimal dataset necessary to show your problem. Creating an effective reprex is a learned skill and the immediate feedback from reprex makes this very concrete.
 
 ## Where to ask
 
-<img src="../help-is-on-the-way.jpg" width="200" height="200" align="right" style="padding:1em;" />
+<img src="../help-is-on-the-way.jpg" alt="Help is on the way elevator button" width="200" height="200" align="right" style="padding:1em;" />
```
Comment (hadley) [inline content/help.md]: Maybe mark this as decoration?

**Comment by batpigandme** [comment]
@hadley changed ✔️

**Comment by hadley** [comment]
http://wave.webaim.org/report#/https://deploy-preview-204--tidyverse-org.netlify.com/packages/ gives a warning about the repeated urls — I wonder if we should just `aria-hidden="true"` the a that contains the image? Or should we `aria-hidden="true"` the learn more link?

**Comment by batpigandme** [comment]
I'm a little confused by the updated audit…it shows the duplicate links, but also that I've added the aria-hidden attribute to them…

**Comment by hadley** [comment]
I suspect that it's just not quite smart enough to figure out that we've dealt with the issue with an aria tag.

**Pull request merged**